tdrs-frontend/src/components/EditProfile/EditProfile.jsx

```diff
@@ -106,13 +119,23 @@ function EditProfile() {
     )
     setErrors(formValidation.errors)
     setTouched(formValidation.touched)
-    setTimeout(() => errorRef.current.focus(), 0)
+
+    if (!Object.values(formValidation.errors).length) {
+      return dispatch(requestAccess(profileInfo))
+    }
+    return setTimeout(() => errorRef.current.focus(), 0)
```

Comment:
This fixes the focus staying on the `Request Access` button even when there are errors per the comment by Robert Jolly. This timeout is necessary because the rerender happens just fast enough that the error section has not appeared on the page, meaning there is nothing to focus on. This fixes that.